### ISSUE TITLE
Set encoding to UTF-8 and Fix Comparison Tests

### ISSUE THREAD

**Issue by darnjo** [RESOStandards/web-api-commander#26]
There were a couple of items caught in pre-review of the Commander for the recent conference demo. Thanks to Trung for finding both of these issues. 
 
- [x] When running Gradle from the Windows command line, but not on the command line in OSX or Linux, and not in IntelliJ in Windows, OSX, or Linux, the response and file streams are not defaulting to UTF-8 and are therefore causing false negatives in validation.

- [x] The comparisons for Integer `ne` aren't taking the possibility of `null` into account. Nulls were previously being filtered by the queries, but the filtering was removed so we could test more of a representative sample of data. The comparisons need to account for this.

**Comment by darnjo** [comment]
Item 1 has been fixed by defaulting to UTF-8, and if we need to add support for other charset encodings, we can. 

Item 2 should be complete, and comparators have been adjusted to better-handle the null case, for instance first checking that `null == null` in the tests that compare for equality. I believe the handling is correct at this point.

**Comment by darnjo** [comment]
I'm going to go ahead and merge this now. Added tests for much of what's there and we should have full coverage now regarding TestUtils and casts/comparisons and null checks, where I found a couple of potential issues when writing the tests and fixed them. See [here for a sample build](https://github.com/RESOStandards/web-api-commander/pull/27/checks?check_run_id=624702105#step:6:33).

```
303 Scenarios (303 passed)
1216 Steps (1216 passed)
0m7.129s
```
